packages/patternlab/styleguide/source/_patterns/05-pages/organization.json

```
@@ -1020,7 +1020,7 @@
                   "eyebrow": "News Article",
                   "title": {
                     "href": "#",
-                    "text": "Tree House Brweing Co. Expands Its Horizons",
+                    "text": "",

```

```suggestion
                    "text": "",
```

Currently backstop is failing. I would recommend adding this text back and remove the condition so match the previous screenshots.